x-pack/plugins/security_solution/server/endpoint/routes/resolver.ts

```diff
@@ -50,6 +50,6 @@ export const registerResolverRoutes = async (
       validate: validateEntities,
       options: { authRequired: true },
     },
-    handleEntities()
+    handleEntities(config.experimentalFeatures)
```

Comment:
maybe we can just send down the desired `experimentalFeature` instead of all of them. What do you think?

Reply:
Make sense to have all the `experimentalFeatures` if we are supposed to add more in the short term. 
Other approach would be having a boolean instead and let the parent functions decide what the boolean has to be. But I'm fine with either approach. Thanks!